src/test/resources/com/puppycrawl/tools/checkstyle/checks/trailingcomment/InputTrailingComment2.java

```diff
@@ -37,8 +37,8 @@ public void run() {
     }
 
     /**
-     * comment with trailing space.
-     */
+     * comment with trailing space.      */
+    // violation below
     final static public String NAME="Some Name"; // NOI18N
```

Comment:
This line is now a violation, since the comment does not conform to the given pattern `^NOI18N$`, due to preceding space.